client/login/controller.js

```diff
@@ -35,6 +36,7 @@ const enhanceContextWithLogin = context => {
 			socialServiceResponse={ context.hash }
 			socialConnect={ flow === 'social-connect' }
 			privateSite={ flow === 'private-site' }
+			domain={ query.domain || null }
```

Comment:
might want to make sure query is not falsey, or default it to an empty object? Not sure how strong the guarantee that the passed `query` will be non-null-or-undefined is.

Reply:
Good point. I added 947eec7ea0 to address that. Thanks.